devdoc/tqueue_requirements.md

```diff
@@ -273,3 +274,21 @@ TQUEUE_POP_RESULT TQUEUE_POP(T)(TQUEUE(T) tqueue, T* item, void* pop_function_co
     - **SRS_TQUEUE_01_045: [** `TQUEUE_POP(T)` shall call `dispose_item_function` with `dispose_item_function_context` as `context` and the array entry value whose state was changed to `POPPING` as `item`. **]**
 
   - **SRS_TQUEUE_01_034: [** `TQUEUE_POP(T)` shall set the state to `NOT_USED` by using `interlocked_exchange`, succeed and return `TQUEUE_POP_OK`. **]**
+
+
+### TQUEUE_GET_COUNT(T)
```

Comment:
>TQUEUE_GET_COUNT(T) [](http://example.com/codeflow?start=4&length=19)

Where can this even be used?

No amount of items in the queue is "real" as in:
step 1: GET_COUNT
step 2: do_something with GET_COUNT... 

between step 1 and step 2 the item count can change. Then how can this information be useful?

I suspect a higher level design errr... "decision" which leads to asking for the number of items in the queue.

Reply:
This API is mostly used for metric and we only need the count as a data point.  
- change name to GET_VOLATILE_COUNT